app/build/page.tsx

```diff
@@ -1,14 +1,20 @@
 "use client";
 
 import 'reactflow/dist/style.css';
-import { ReactFlowProvider } from 'reactflow';
+import {ReactFlowProvider} from 'reactflow';
+import {Suspense} from 'react';
 import Graph from '@/components/build/graph';
-import { GraphContextProvider } from '@/contexts/graph';
+import {GraphContextProvider} from '@/contexts/graph';
+
+export default function Build() {
+    return (
+        <ReactFlowProvider>
+            <GraphContextProvider>
+                <Suspense>
```

Comment:
I added this to fix [one of the linting errors](https://nextjs.org/docs/messages/missing-suspense-with-csr-bailout) I encountered building prod.

From the docs:

> This will ensure the page does not de-opt to client-side rendering.